pyglmnet/datasets.py

```diff
@@ -43,45 +41,73 @@ def _reporthook(count, block_size, total_size):
                      % (percent, progress_size / (1024 * 1024)))
 
 
-def fetch_tikhonov_data(dpath):
+def get_data_home(data_home=None):
+    """Return the path of the pyglmnet data dir.
+
+    Function from scikit-learn.
+
+    This folder is used by some large dataset loaders to avoid downloading the
+    data several times.
+
+    By default the data dir is set to a folder named 'glm_data' in the
+    user home folder.
+
+    Parameters
+    ----------
+    data_home : str | None
+        The path to pyglmnet data dir.
     """
-    Downloads data for Tikhonov example and returns data frames
+    if data_home is None:
+        data_home = op.join('~', 'glm_data')
+    data_home = op.expanduser(data_home)
+    if not op.exists(data_home):
+        os.makedirs(data_home)
+    return data_home
+
+
+def fetch_tikhonov_data(dpath=None):
+    """Downloads data for Tikhonov example.
 
     Parameters
     ----------
     dpath: str
         specifies path to which the data files should be downloaded.
+        default: None
```

Comment:
I normally don't bother with defaults because you can see it by using python's introspection abilities.

Do:

```py
>>> fetch_tikhonov_data?
```